Bundle/FormBundle/Resources/public/js/link.js

```diff
@@ -1,15 +1,29 @@
+/**
+ * This function is use to display or hide link fields.
+ * If the hidden input "current" is not null, this function show the good fields
+ * This hidden input has been added in the linkType because when the "data-refreshOnChange" was used,
+ * link field took 'none link' value, or value shouldn't be change.
+ */
 function showSelectedLinkType(selects) {
```

Comment:
This is not the right way to get it work... Actually, you just need to be sure the showSelectedLinkType function is well called when the form is reloaded